docs/en/02_Developer_Guides/03_Forms/Field_types/03_HTMLEditorField.md

```diff
@@ -185,8 +185,8 @@ To disable oEmbed usage, set the `Oembed.enabled` configuration property to "fal
 HtmlEditorField can have whitelists set on both the scheme (default http & https) and domains allowed when
 inserting files for use with oembed.
 
-This is performed through the config variables `HtmlEditorField_Toolbar::$fileurl_scheme_whitelist` and
-`HtmlEditorField_Toolbar::$fileurl_domain_whitelist`.
+This is performed through the config variables `ModalController::$fileurl_scheme_whitelist` and
+`ModalController::$fileurl_domain_whitelist`.
```

Comment:
I think these config settings were moved to `RemoteFileFormFactory` rather than `ModalController`